PublicationsRouter/oai-pmh/RIOXX_XWALK.md

```diff
@@ -0,0 +1,89 @@
+# PubRouter Notification to OAI-RIOXX Crosswalk
+
+The table below lists the RIOXX terms that are provided by PubRouter's implementation of OAI-PMH, and shows how PubRouter's metadata elements are mapped to these.
```

Comment:
not PubRouter's implentation of OAI-PMH, it's the implementation of a rioxx metadata type